### ISSUE TITLE
Server list is blank

### ISSUE THREAD

**Issue by BabRoosss** [Heath123/discross#42]
I have only tested on the most recent version of the Wii Browser and modern Firefox, but the server list is blank. I input the code I got from my server, but the list is blank.

**Comment by CauseIWont** [comment]
Same. I hope they're working on a fix soon-ish.

**Comment by chasio2221** [comment]
I have this error on chrome

**Comment by AvaGoosa** [comment]
Bump! I am having this "glitch" too. Even after deleting/resetting my account (as I heard discriminators being removed may break it) I still have an empty list. I have re-paired it many times and it's still blank. I've tested on an Xbox 360, a Wii, Safari on iOS 9 and 15, and Chrome on Windows and Linux, so I don't believe this is a device-specific problem. Hope that helps!

**Comment by larsenv** [comment]
@BabRoosss @CauseIWont @chasio2221 @Fox595676 I know this issue is old, but the issue is fixed. Can you try it

**Comment by GamerGator665** [comment]
@larsenv I'm not so sure about that! Sorry to bump but I'm having this issue and I just signed up!

**Comment by GamerGator665** [comment]
> @larsenv I'm not so sure about that! Sorry to bump but I'm having this issue and I just signed up!

I say just, it was like 2 or 3 days ago.

**Comment by larsenv** [comment]
@GamerGator665 did you authorize discross?

**Comment by BabRoosss** [comment]
> @BabRoosss @CauseIWont @chasio2221 @Fox595676 I know this issue is old, but the issue is fixed. Can you try it?

sorry for the late response, i forgot i made this lol. yes, the issue has been fixed.

**Comment by GamerGator665** [comment]
> @GamerGator665 did you authorize discross?

Yes I just realised I had to do that. I signed in on my wii and didn't see any servers. I've now authorised using my phone and I wasn't sure if you needed to do that.

**Comment by larsenv** [comment]
@GamerGator665 make sure you are in a discord that has discross

**Comment by AvaGoosa** [comment]
> Bump! I am having this "glitch" too. Even after deleting/resetting my account (as I heard discriminators being removed may break it) I still have an empty list. I have re-paired it many times and it's still blank. I've tested on an Xbox 360, a Wii, Safari on iOS 9 and 15, and Chrome on Windows and Linux, so I don't believe this is a device-specific problem. Hope that helps!

For me it's fixed now! After they brought Discross back, I linked my account again and after a few tries, it works now. Solved on my end :)
